index.js

```diff
@@ -25,6 +25,10 @@ http.request = function (opts, cb) {
 	opts.hostname = opts.hostname || hostHostname || window.location.hostname
 	opts.port = opts.port || hostPort || defaultPort
 
+	if (opts.path.indexOf('./') == 0) {
+		opts.path = window.location.pathname.slice(0, window.location.pathname.lastIndexOf('/')) + opts.path.slice(1)
```

Comment:
Hm, I think there's got to be a cleaner way of doing this; at the minimum we should use `url.parse` instead of string manipulation.